### PR TITLE
Use stylelint-config-standard

### PR DESCRIPTION
**Pull request by xPaw**
We're surprisingly quite standard except for three rules.

### CODE REVIEW

File: client/css/style.css [hunk 1]
```diff
@@ -1722,10 +1721,7 @@ kbd {
 	border-right-color: #222;
 }
 
-@media only screen and (-webkit-min-device-pixel-ratio: 2),
```
Comment (YaManicKill) [inline client/css/style.css]: Are these no longer needed?

File: .editorconfig [hunk 2]
```diff
@@ -6,6 +6,7 @@ root = true
 
 [*]
 indent_style = tab
+tab_width = 4
```
Comment (YaManicKill) [inline .editorconfig]: Such big tabs. Weird people.

Comment (xPaw) [inline .editorconfig]: This was mainly for github (it uses editor config), and it defaults to 8 tabs.

File: client/css/style.css (hunk 1 above)
Comment (xPaw) [inline client/css/style.css]: `resolution` is the way to do it, min-device-pixel-ratio is deprecated.

File: .editorconfig (hunk 2 above)
Comment (YaManicKill) [inline .editorconfig]: Alright, 4 is definitely better than 8.

Comment (astorije) [inline .editorconfig]: Why not 2 to match L20 and former L24?

**Comment by xPaw** [comment]
Removed `tab-width` from this PR.

**Pull request merged**